src/averageAges.js

```diff
@@ -37,7 +59,24 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  if (withChildren === undefined) {
+    const arrOfWomens = people.filter(item => item['sex'] === 'f');
+    const sumOfWomensAge = arrOfWomens.reduce((startValue, person) => {
+      return startValue + (person['died'] - person['born']);
```

Comment:
```suggestion
      return startValue + (person.died - person.born);
```
why do you complicate this?)